src/api/common_api.js

```diff
@@ -811,11 +811,12 @@ module.exports = {
         },
         nsfs_account_config: {
```

Comment:
Do we want to also let nsfs_only be configured via update_account_s3_access and delete_account_by_property?
If so then that RPC is not using this type.
Please make sure to go across all subtypes of that and verify that we do not want to add this capability to them as well.

Reply:
added nsfs_only to update_account_s3_access api but not to delete_account_by_property